sentences/pl/weather_HassGetWeather.yaml

```diff
@@ -0,0 +1,10 @@
+language: pl
+intents:
+  HassGetWeather:
+    data:
+      - sentences:
+          - "<what_is> pogoda"
+      - sentences:
+          - "<what_is> pogoda (w|dla) <name>"
```

Comment:
Please add extra word `we` .. correct sentence is say: "Jaka jest pogoda we Wrocławiu" 😀

Reply:
done